src/simularium/VisGeometry/PDBModel.ts

```diff
@@ -285,13 +304,48 @@ class PDBModel {
 
         // update the new LODs
         for (let i = 0; i < sizes.length; ++i) {
-            this.lods[i + 1] = {
-                geometry: this.createGPUBuffer(retData[i]),
+            const lodIndex = i + 1;
+            // if old LOD existed we can dispose now.
+            if (this.lods[lodIndex]) {
+                this.lods[lodIndex].instances.dispose();
+            }
+            const geometry = this.createGPUBuffer(retData[i]);
+            this.lods[lodIndex] = {
+                geometry: geometry,
                 vertices: retData[i],
+                instances: new InstancedMesh(
+                    InstanceType.POINTS,
+                    geometry,
+                    this.name + "_LOD" + lodIndex,
+                    0
+                ),
             };
         }
     }
 
+    public numLODs(): number {
+        return this.lods.length;
+    }
+
+    public getLOD(index: number): InstancedMesh {
```

Comment:
I find it confusing that this is named `getLOD`, it sounds like a method that gets a level of detail which I think of as a number or something, but it actually returns an `InstancedMesh` with a specific level of detail. Is that conventional?

Reply:
I would say this is not too out of the ordinary - a level of detail is a representation of the object. (And obviously we are passing the number in.)  Would you be more comfortable if it was something like `getLODObject3D` or getMeshLOD?

Reply:
Okay. If it's just me being new to this, and it's not confusing to others, I'm cool with `getLOD`.